libs/velo-external-db-core/src/spi-model/collection.ts

```diff
@@ -0,0 +1,165 @@
+
```

Comment:
Not really important, but if you can delete the extra lines here, and put a line between every type/interface in this file.